src/duration.js

```diff
@@ -770,7 +780,25 @@ export default class Duration {
 
   /**
    * Convert this Duration into its representation in a different set of units.
-   * @example Duration.fromObject({ hours: 1, seconds: 30 }).shiftTo('minutes', 'milliseconds').toObject() //=> { minutes: 60, milliseconds: 30000 }
+   *
+   * Note that using shiftTo() can sometimes have unexpected results when using units that don't refer to a
```

Comment:
Here too. Now like 95% of the doc for this is a troubleshooting guide for conversionAccuracy. I think here we just say:

> Converting between units is subtle because units don't always have constant length. Please see https://moment.github.io/luxon/#/math?id=casual-vs-longterm-conversion-accuracy